docs/pc-and-console/helpshift-apis.mdx

```diff
@@ -199,6 +199,17 @@ What extra information should be included in the response. A JSON array of strin
 
 - unread_messages - if the user has any active conversations, the API response will<br />contain the issue ID and number of unread messages.
 
+</td>
+</tr>
+<tr>
+<td>initiate_chat</td>
+<td>No</td>
+<td>
+Whether to start a new chat when the previous issue is resolved. Valid values -
+
+- true:  a new chat will be started automatically when the previous issue is resolved and without needing any user action like clicking the New Conversation button or going through the post resolution flow like feedback bot for the previously resolved issue.
```

Comment:
`New Conversation`

Code formatted since it is a CTA.

Reply:
Done